### DISCUSSION TITLE
LLDP implementation under STM32F and LwIP environment

### DISCUSSION THREAD

**Discussion by virennshah-eaton** [lldpd/lldpd#548] [Q&A]
Hi All,
I am trying to implement LLDP under STM32F7XX controller and LwIP library environment.
Could you help with any guideline/document that will help to achieve this?

Thanks!

**Comment by vincentbernat** [comment]
LLDP does not use an IP stack. It works on some MMU-less systems by using `vfork()` instead of `fork()`. However, it may be too big for a microcontroller.
